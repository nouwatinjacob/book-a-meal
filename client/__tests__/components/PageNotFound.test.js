import React from 'react';
import {
  shallow
} from 'enzyme';
import PageNotFound from '../../src/components/PageNotfound.jsx';

describe('PageNotFound component', () => {
  const setup = () => shallow(<PageNotFound />);

  it('should render PageNotFound component correctly', (done) => {
    const wrapper = setup();
    expect(wrapper).toMatchSnapshot();

    done();
  });
});