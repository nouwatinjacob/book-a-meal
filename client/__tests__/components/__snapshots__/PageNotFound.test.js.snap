// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound component should render PageNotFound component correctly 1`] = `
<Fragment>
  <div
    className="pageNotFound"
  >
    <h5
      className="text-center"
    >
      Click 
      <a
        href="/menus"
      >
        here
      </a>
       to go to the menus page
    </h5>
  </div>
</Fragment>
`;
